dataset/data.go

```diff
@@ -6,33 +6,40 @@ import (
 	"unicode"
 )
 
-// Model represents a response dataset model from the dataset api
-type Model struct {
-	ID                string           `json:"id"`
-	CollectionID      string           `json:"collection_id"`
-	Contacts          []Contact        `json:"contacts"`
-	Description       string           `json:"description"`
-	Keywords          []string         `json:"keywords"`
-	License           string           `json:"license"`
-	Links             Links            `json:"links"`
-	Methodologies     []Methodology    `json:"methodologies"`
-	NationalStatistic bool             `json:"national_statistic"`
-	NextRelease       string           `json:"next_release"`
-	Publications      []Publication    `json:"publications"`
-	Publisher         *Publisher       `json:"publisher"`
-	QMI               Publication      `json:"qmi"`
-	RelatedDatasets   []RelatedDataset `json:"related_datasets"`
-	ReleaseFrequency  string           `json:"release_frequency"`
-	State             string           `json:"state"`
-	Theme             string           `json:"theme"`
-	Title             string           `json:"title"`
-	UnitOfMeasure     string           `json:"unit_of_measure"`
-	URI               string           `json:"uri"`
+// DatasetDetails represents a response dataset model from the dataset api
+type DatasetDetails struct {
+	ID                string           `json:"id,omitempty"`
+	CollectionID      string           `json:"collection_id,omitempty"`
+	Contacts          []Contact        `json:"contacts,omitempty"`
```

Comment:
Any arrays which are not pointers (e.g. `*[]Contact`) will be returned as empty arrays in the json instead of not appearing at all. Thought I'd mention it as I don't know whether you mean to have empty arrays returned in the response?

Reply:
👍